nautobot/docs/release-notes/version-1.5.md

```diff
@@ -148,6 +148,28 @@ A number of mixin classes have been renamed and/or relocated for improved self-c
 | `TenancyFilterSet`             | `TenancyModelFilterSetMixin`                 |
 
 <!-- towncrier release notes start -->
+## v1.5.8 (2023-01-23)
+
+### Added
+
+- [#3103](https://github.com/nautobot/nautobot/issues/3103) - Added Redis troubleshooting section to installation docs.
+
+### Changed
+
+- [#3055](https://github.com/nautobot/nautobot/issues/3055) - Update `setuptools` to `65.5.1` to address `CVE-2022-40897`.
+- [#3072](https://github.com/nautobot/nautobot/issues/3072) - In Nautobot's unit tests, all HTTP requests are now sent with SERVER_NAME set to `nautobot.example.com` instead of `testserver` (Django's default) and the test configuration for Nautobot itself sets `ALLOWED_HOSTS` to expect `nautobot.example.com`. This is intended to protect against issues such as #3065.
+- [#3077](https://github.com/nautobot/nautobot/issues/3077) - Updated Nautobot release checklist to reflect current branching and pull request process.
+- [#3082](https://github.com/nautobot/nautobot/issues/3082) - Updated `gitpython` to `~3.1.30` to address CVE-2022-24439.
```

Comment:
```suggestion
- [#3082](https://github.com/nautobot/nautobot/issues/3082) - Updated `gitpython` to `~3.1.30` to address `CVE-2022-24439`.
```

Reply:
Also, should 3055, 3082, and 3119 be listed under `Security` rather than `Changed`?